buildSrc/build.gradle

```diff
@@ -1,13 +1,19 @@
 plugins {
 	id 'java-gradle-plugin'
 	id 'checkstyle'
+	id 'io.spring.javaformat' version "${javaFormatVersion}"
 }
 
 repositories {
 	mavenCentral()
 	gradlePluginPortal()
 }
 
+tasks.named('compileJava') {
```

Comment:
how to bind this properly, please?

only the `checkFormat` goal is executed by default, but not the `format`.

- https://github.com/spring-io/spring-javaformat?tab=readme-ov-file#gradle